mftp/ntfy.py

```diff
@@ -0,0 +1,80 @@
+import re
+import requests
+from bs4 import BeautifulSoup as bs
+import logging
+from notice import update_lsni
+from endpoints import NOTICE_CONTENT_URL
+
+def format_notice(notices, session):
+  notifications=[]
+
+  for notice in reversed(notices):
+    id_, year = notice['UID'].split('_')
+
+    try:
+      data = parseBody(notice, session, year, id_)
+      body, links = parseLinks(data, id_)
+    except Exception as e:
+      logging.error(f"Failed to parse notification body ~ {str(e)}")
+
+    if notice['Subject'] == 'Urgent':
+      priority="5"
+    else:
+      priority="4"
+
+    notifications.append(
+      {"Title":  f"#{id_} | {notice['Type']} | {notice['Subject']} | {notice['Company']}",
+        "Body": body,
+        "Tags": f"{notice['Type']}, {notice['Subject']}, {notice['Company']}",
+        "Priority": priority,
+        "Links": links})
+  
+  return notifications
+
+def send(notifications, lsnif, notices):
+  for i, notification in enumerate(notifications, 1): 
+    try:
+      requests.post("http://172.18.0.2:8000/mftp",
+        data=notification["Body"],
+        headers={ 
+          "Title": notification["Title"],
+          "Tags": notification["Tags"],
+          "Priority": notification["Priority"],
+          "Icon": "https://miro.medium.com/v2/resize:fit:600/1*O94LHxqfD_JGogOKyuBFgA.jpeg",
+          "Action": notification["Links"]})
+      update_lsni(lsnif, notices, i)
+      print(f'Sent notification {notification["Title"]}')
```

Comment:
https://github.com/metakgp/mftp/blob/fea17c2bd9c0230b2621fcb83c17afed0e084e30/mftp/mail.py#L59

Adhere with the previous codebase, how it was implemented shouldn't change for code consistency.